btx/misc/shortcuts.py

```diff
@@ -29,3 +31,27 @@ def __getattr__(self, item):
             return self.__getitem__(item)
         except KeyError:
             raise AttributeError(item)
+
+def fetch_latest(fnames, run):
```

Comment:
Note for later: this could potentially go with another function tasked with defining paths to mrxv, etc. based on the location (SLAC, NERSC, ...). Maybe in some `path_interface` tool, or something